### PR TITLE
1.9.6 changelog

### PR DESCRIPTION
**Pull request by jamiedemaria**

### CODE REVIEW

**Comment by jamiedemaria** [comment]
* **#26555** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/26555?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈 <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/26555?utm_source=stack-comment-view-in-graphite" target="_blank">(View in Graphite)</a>
* `master`


<h2></h2>

This stack of pull requests is managed by <a href="https://graphite.dev?utm-source=stack-comment"><b>Graphite</b></a>. Learn more about <a href="https://stacking.dev/?utm_source=stack-comment">stacking</a>.

File: CHANGES.md [hunk 1]
```diff
@@ -1,5 +1,38 @@
 # Changelog
 
+## 1.9.6 (core) / 0.25.6 (libraries)
+
+### New
+
+- Added Google Drive icon for `kind` tags. (Thanks, [@dragos-pop](https://github.com/dragos-pop)!)
```
Comment (OwenKephart) [inline CHANGES.md:7]: I think this counts as [ui]

File: CHANGES.md [hunk 2]
```diff
@@ -1,5 +1,38 @@
...
+- [helm] Added the ability to set user-deployments labels on k8s deployments as well as pods.
+
+### Bugfixes
+
+- Assets with self dependencies and `BackfillPolicy` are now evaluated correctly during backfills. Self dependent assets no longer result in serial partition submissions or disregarded upstream dependencies.
+- Previously, the freshness check sensor would not re-evaluate freshness checks if an in-flight run was planning on evaluating that check. Now, the freshness check sensor will kick off an independent run of the check, even if there's already an in flight run, as long as the freshness check can potentially fail.
+- Previously, if the freshness check was in a failing state, the sensor would wait for a run to update the freshness check before re-evaluating. Now, if there's a materialization later than the last evaluation of the freshness check and no planned evaluation, we will re-evaluate the freshness check automatically.
+- [ui] Fix run log streaming for runs with a large volume of logs.
```
Comment (OwenKephart) [inline CHANGES.md:26]: Fix -> Fixed

**Review by OwenKephart — APPROVED** [review]
note comments

**Pull request merged**